content/100-getting-started/02-setup-prisma/100-start-from-scratch/110-relational-databases/150-using-prisma-migrate.mdx

```diff
@@ -52,6 +71,35 @@ This command does two things:
 
 > **Note**: `generate` is called under the hood by default, after running `prisma migrate dev`. If the `prisma-client-js` generator is defined in your schema, this will check if `@prisma/client` is installed and install it if it's missing.
 
+</SwitchTech>
+
+<SwitchTech technologies={['*', 'planetscale']}>
+
+You are now ready to push your new schema to the `develop` branch of your database. To do this, use the `db push` CLI command:
```

Comment:
We do not have concrete instructions before this how to really connect to the `develop` branch (or create the branch, or create a special connection string for it) before this is mentioned. Right now the user is probably connected to their main, production branch and will get an error message instead.

We are also missing instruction on how to merge the schema changes to the production branch later to be able to use it and not just having them in `develop` with no way to keep them around in the main, production branch.

Reply:
I'm still unclear about how much database-specific detail should be handled in these getting started pages and how much should be assumed that the user already knows. The PostgreSQL, MySQL etc pages do not go into too much detail, but maybe that is just because there isn't too much detail required beyond the connection string. PlanetScale has significantly more because of the branching, but I don't know whether I should be duplicating their instructions on how it works here. But yes, I can add this all in.

Reply:
I've made an attempt to add in this information, let me know if it looks correct.

Reply:
Hm, deploy is failing because of my link back to the PlanetScale/TypeScript version of 'Connect your database'. Is there something special I need to do with links to these tech switcher pages?

Reply:
I think our getting started guide should indeed include all the information required to really just follow the instructions and then have a successful experience - in this case that means pointing out that schema changes can only happen on a branch, that you run `db push` (instead of other migrate commands), and that you need to merge the schema changes back to the production branch before you write stuff into your database (or it will go away together with the branch). Then at the end the user should have learned all the things they need to know about using Prisma with PlanetScale. 

(For normal database providers without a custom branch and merge workflow for schema changes, all that is indeed not necessary at all - but that is the "cost" (and benefit) of using PlanetScale)

Reply:
Replied to the link problem via comment on the commit that added the link: [`3d7e843` (#2625)](https://github.com/prisma/docs/pull/2625/commits/3d7e843cbf661991af8a4b0d3086dd6da46c01a7#r773847509)